ja_JP/docs/recipes/code-coverage.md

```diff
@@ -27,11 +42,19 @@ ES5をカバーするには、単純にテストスクリプトの先頭に`nyc`
 }
 ```
 
+これで完了です。
+
+もしHTMLのカバレッジレポートが欲しい場合や、カバレッジデータをCoverallsにアップロードしたい場合なら、下のセクションはスキップしてください。
```

Comment:
To be honest, I don' know that what is the correct translation for `skip down`. But `スキップしてください` is clearly meaning of the opposite because what this paragraph mean is `If you want to create HTML coverage or upload coverage data, read the section below that mensions about coverage.`

I think it should be translated as `HTMLのカバレッジレポートを作りたい場合やCoverallsにカバレッジデータをアップロードしたい場合、以下のセクションを参照してください。`